### DISCUSSION TITLE
NoneBot 文档 框架/主题 征集

### DISCUSSION THREAD

**Discussion by yanyongyu** [nonebot/discussions#21] [想法]
## Purpose

由于历史遗留问题，`nonebot` 一直采用 `vuepress` 作为文档框架。并且由于开始时的文档管理不规范、没有一个合适的文档发布流程，导致目前出现了不少管理与浏览体验上的问题。

现向社区征集建议，希望有合适的文档框架以及与之配合的主题来完善文档浏览体验。

## Detail

我们希望文档能满足以下基本要求：

- 支持 markdown 展示
- 具有美观的界面

支持以下功能更佳：

- 支持 PWA
- 浏览不卡顿 / 具有一定的动画
- 支持文档版本切换
- 可以自定义主题或部分组件

**Comment by stdrc** [comment]
推荐个 mkdocs

**Comment by yanyongyu** [comment]
[docusaurus](https://docusaurus.io)

**Reply by yanyongyu** [reply]
目前使用docusaurus尝试编写新主题

由于docusaurus插件支持多版本文档构建，便于管理文档版本

**Comment by yanyongyu** [comment]
vuepress/vitepress （新写一个theme）
